narwhals/expr.py

```diff
@@ -1175,11 +1194,57 @@ def fill_null(self, value: Any) -> Self:
             │ 2   ┆ 2.0 │
             │ 4   ┆ 4.0 │
             │ 0   ┆ NaN │
+            │ 0   ┆ NaN │
             │ 3   ┆ 3.0 │
             │ 5   ┆ 5.0 │
             └─────┴─────┘
+
+            Using a strategy:
+
+            >>> @nw.narwhalify
+            ... def func_strategies(df):
+            ...     return df.with_columns(
+            ...         nw.col("a", "b").fill_null(strategy="forward", limit=1)
+            ...     )
+
+            >>> func_strategies(df_pd)
+                 a    b
+            0  2.0  2.0
+            1  4.0  4.0
+            2  4.0  4.0
+            3  NaN  NaN
+            4  3.0  3.0
+            5  5.0  5.0
+
+            >>> func_strategies(df_pl)  # nan != null for polars
```

Comment:
it's a bit hard to see what's changed...how about doing `nw.col("a", "b").fill_null(strategy="forward", limit=1).name.suffix('_filled')`?

Reply:
Done!